packages/trader/src/App/Components/Elements/ContractDrawer/contract-drawer.tsx

```diff
@@ -64,6 +64,9 @@ const ContractDrawer = observer(
         const contract_drawer_ref = React.useRef<HTMLDivElement>(null);
         const contract_drawer_card_ref = React.useRef<HTMLDivElement>(null);
         const [should_show_contract_audit, setShouldShowContractAudit] = React.useState(false);
+
+        const MARGIN_BOTTOM_SIZE = 13;
```

Comment:
Hmm... It's just used once across this file, why do we need to make a new variable for this? It just creates an unnecessary need to go here and there through the code to try to find where this value comes from, whenever someone else tries to change it. Moreover, it's not like we can just open the file, see and change it, first it's not even in the beginning of the file, plus the name doesn't really tell us much about the meaning of the number. Let's stop creating unutilized variables in the future 👍🏻

Reply:
This constant was created in order to get rid of 'magic number'. What is 13? Why exactly 13? So many questions
I red, that it's a good practise to hold such values in constants with short, intuitive and descriptive naming. 
I can lift the constant up (to the very begging of the file) and rename it if you think it doesn't tell much about the meaning, but let's ask for others opinion too